### ISSUE TITLE
Examples to ChatGPT

### ISSUE THREAD

**Issue by un4rch** [un4rch/ChatTFG#14]
AS A Student

I WANT to ask for examples

SO THAT I understand better

**Comment by xabier** [comment]
I don't think we need this to be a feature, it is a default feature. Feel free to remove it, I have included it on the MVP.
